### PR TITLE
Edit README to follow the NOS template

### PR DESCRIPTION
**Pull request by xrigau**
I also added the missing LICENSE file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,9 +28,18 @@ To get the front-end running do:
  - `ng serve`
  - Open a browser `http://localhost:4200/dashboards`
 
-To get the android app complied do:
+To get the android app running do:
```
Comment (mr-archano) [inline README.md]: Isn't `installDebug` only installing the apk? If you really want to run it maybe it should be `runDebug` right?

File: README.md [hunk 2]
```diff
@@ -24,9 +28,18 @@ To get the front-end running do:
...
\ No newline at end of file
+ 
+## Links
+
+Here are a list of useful links:
+
+ * We always welcome people to contribute new features or bug fixes, [here is how](https://github.com/novoda/novoda/blob/master/CONTRIBUTING.md)
+ * If you have a problem check the [Issues Page](https://github.com/novoda/dashboards/issues) first to see if we are working on it
+ * For further usage or to delve more deeply checkout the [Project Wiki](https://github.com/novoda/dashboards/wiki)
+ * Looking for community help, browse the already asked [Stack Overflow Questions](http://stackoverflow.com/questions/tagged/support-dashboards) or use the tag: `support-dashboards` when posting a new question
```
Comment (mr-archano) [inline README.md]: Does it mean you created the tags in SO as well?

Comment (xrigau) [inline README.md]: not sure what you mean, afaik you can just create a new question with that tag without having to create it beforehand

File: README.md (hunk 1 above)
Comment (xrigau) [inline README.md]: that task doesn't exist in the android gradle plugin, that's why there's an extra line saying you have to run the app manually 
https://github.com/novoda/dashboards/pull/7/files/2065657dd3f5eecfe61bc83c3edac794ee503066#diff-04c6e90faac2675aa89e2176d2eec7d8R35

**Comment by florianmski** [comment]
LGTM 👍 
Not sure how to merge it though 🎱

File: README.md (hunk 1 above)
Comment (mr-archano) [inline README.md]: Well, it exists in the [android command plugin](https://github.com/novoda/gradle-android-command-plugin) :trollface: (My bad, I was of the idea you were already using it in the sample app).

**Comment by xrigau** [comment]
Retest this

**Comment by xrigau** [comment]
Should be good to go once PRB succeeds @florianmski 💃

**Comment by florianmski** [comment]
:shipit:

**Pull request merged**